src/servers/Cargo.toml

```diff
@@ -41,6 +41,7 @@ derive_builder = "0.12"
 digest = "0.10"
 futures = "0.3"
 hex = { version = "0.4" }
+hostname = "0.3.1"
```

Comment:
I've checked its [implementation](https://github.com/svartalf/hostname/blob/master/src/nix.rs#L11-L30). It uses the `gethostname` syscall and has no unwrap 👍